problem_builder/templates/html/mentoring-with-steps-add-buttons.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+<div class="add-xblock-component new-component-item adding">
+  <div class="new-component">
+    <h5>{% trans "Add New Component" %}</h5>
+    <ul class="new-component-type">
+      <li><a href="#" class="single-template add-xblock-component-button" data-category="pb-mentoring-step">{% trans "Mentoring Step" %}</a></li>
+      <li><a href="#" class="single-template add-xblock-component-button" data-category="pb-message" data-boilerplate="completed">{% trans "Message (Complete)" %}</a></li>
+      <li><a href="#" class="single-template add-xblock-component-button" data-category="pb-message" data-boilerplate="incomplete">{% trans "Message (Incomplete)" %}</a></li>
+      <li><a href="#" class="single-template add-xblock-component-button" data-category="pb-message" data-boilerplate="max_attempts_reached">{% trans "Message (Max # Attempts)" %}</a></li>
+      <li><a href="#" class="single-template add-xblock-component-button" data-category="pb-message" data-boilerplate="on-assessment-review">{% trans "Message (Assessment Review)" %}</a></li>
```

Comment:
@itsjeyd not needed if MentoringWithSteps uses `StudioContainerWithNestedXBlocksMixin`